packages/shared/lib/integrations/scripts/webhook/webhook.manager.ts

```diff
@@ -10,7 +10,7 @@ import type { WebhookHandlersMap, WebhookResponse } from './types.js';
 
 const handlers: WebhookHandlersMap = webhookHandlers as unknown as WebhookHandlersMap;
 
-async function execute(environmentUuid: string, providerConfigKey: string, headers: Record<string, any>, body: any, rawBody: string): Promise<void | any> {
+async function execute(environmentUuid: string, providerConfigKey: string, headers: Record<string, any>, body: any, rawBody: string): Promise<any> {
```

Comment:
looks to me that it can actually return void

Reply:
eslint is complaining about that because `void` should only be used when there is no return value, so it can't be `void or any` in that case it's `undefined | any` and truthfully it's `any` :D
https://typescript-eslint.io/rules/no-invalid-void-type/